addons/project/views/project_project_views.xml

```diff
@@ -129,7 +129,18 @@
                                 </group>
                             </group>
                             <group>
-                                <group name="group_tasks_managment" string="Tasks Management" col="1" class="row mt16 o_settings_container" groups="project.group_project_task_dependencies,project.group_project_milestone,project.group_project_recurring_tasks">
+                                <!-- TODO: Remove in master -->
+                                <group name="group_tasks_managment" string="Tasks Management" col="1" class="row mt16 o_settings_container d-none" groups="project.group_project_task_dependencies,project.group_project_milestone,project.group_project_recurring_tasks">
+                                    <div>
+                                        <setting class="col-lg-12" id="task_dependencies_setting" help="Determine the order in which to perform tasks" groups="project.group_project_task_dependencies">
+                                            <field name="allow_task_dependencies"/>
+                                        </setting>
+                                        <setting class="col-lg-12" id="project_milestone_setting" help="Track major progress points that must be reached to achieve success" groups="project.group_project_milestone">
+                                            <field name="allow_milestones"/>
+                                        </setting>
+                                    </div>
+                                </group>
```

Comment:
We need to remove this since any customization in this section could make it hidden.